backend/src/providers/analytics/posthog.provider.ts

```
@@ -66,6 +77,7 @@ export class CloudPosthogProvider implements PosthogProvider {
   }
 
   async getConnection(): Promise<PosthogConnection | null> {
+    if (!this.isEnabled()) this.throwUnsupported();
```

ESLint

[error] Expected { after 'if' condition.

(curly)